libs/backend/src/cast_vote_records/build_cast_vote_record.ts

```diff
@@ -55,6 +55,35 @@ export function toCdfBallotType(ballotType: BallotType): CVR.vxBallotType {
   }
 }
 
+/**
+ * The input to {@link buildCvrImageData}
+ */
+export interface CvrImageDataInput {
+  imageHash: string;
+  imageRelativePath: string;
+  layoutFileHash?: string;
```

Comment:
Optional because, even when exporting images, we don't export layout files for BMD ballots

Reply:
Thanks for the note. Might it be better as a code comment?

Reply:
Good call: https://github.com/votingworks/vxsuite/pull/4056/commits/9072ae191f935d2a4a15886d0be6f419ea1e3fcf